setup.py

```diff
@@ -6,20 +6,21 @@
     long_description = fh.read()
 
 requirements = [
-    "sympy>=1.10.1",
-    "numpy==1.22.3",
-    "networkx==2.8",
-    "scipy==1.8",
+    "amazon-braket-sdk==1.23.0",
```

Comment:
Before merging we need to make sure the dependencies are ok. Sorry for the previous mess!